src/Illuminate/Foundation/Testing/TestResponse.php

```diff
@@ -494,9 +494,10 @@ public function assertJsonMissingExact(array $data)
      *
      * @param  array|null  $structure
      * @param  array|null  $responseData
+     * @param  bool        $exact
      * @return $this
      */
-    public function assertJsonStructure(array $structure = null, $responseData = null)
+    public function assertJsonStructure(array $structure = null, &$responseData = null, bool $exact = false)
```

Comment:
Why are we mutating the data?

Reply:
You are right, we can achieve this without mutating the data.